src/AutoRest.CSharp/Mgmt/Decorator/ResourceDetection.cs

```diff
@@ -110,9 +109,9 @@ private static bool CheckEvenSegments(string requestPath)
             return segments.Length % 2 == 0;
         }
 
-        private static bool TryOperationWithMethod(this OperationSet set, HttpMethod method, [MaybeNullWhen(false)] out ObjectSchema resourceSchema)
+        private static bool TryOperationWithMethod(this OperationSet set, RequestMethod method, [MaybeNullWhen(false)] out InputModelType resourceType)
```

Comment:
resourceType might be misunderstood in the context of MPG, could we change to `resourceModel` here?

Reply:
renamed to `inputModel`